Friendlier message when no bind addresses

huntc opened the pull request:
As this error always appears for first time users, we should present it in a friendlier fashion. This commit attempts to do that.

huntc committed:
Friendlier message when no bind addresses

As this error always appears for first time users, we should present it in a friendlier fashion. This commit attempts to do that.

huntc commented:
Sample output:

```
ChristohersMBP2:conductr-cli huntc$ sandbox run 2.0.0-rc.3 -n1:3 -f visualization -f logging -f monitoring
|------------------------------------------------|
| Starting ConductR                              |
|------------------------------------------------|
|------------------------------------------------|
| Whoops. Network address aliases are required   |
| so that the sandbox can operate as a cluster   |
| of machines                                    |
|------------------------------------------------|

Please run the following and then try your command again:

sudo ifconfig lo0 alias 192.168.10.1 255.255.255.0
sudo ifconfig lo0 alias 192.168.10.2 255.255.255.0
sudo ifconfig lo0 alias 192.168.10.3 255.255.255.0
```

markusjura commented on conductr_cli/host.py:
```diff
@@ -81,10 +82,9 @@ def is_listening(ip_addr, port):
-            return 'Run the following commands to create the address aliases on your machine:'
-        else:
-            return 'Run the following command to create an address alias on your machine:'
+        return headline(
```
I would not use a headline here. Headlines should we used for certain sub sections to start ConductR, such as `Starting HAProxy`.

huntc replied:
Ok but I think it needs to standout somehow

markusjura replied:
I am wondering if we should add the address aliases automatically after the user entered the sudo password. When prompting for the password, we could explain what we are doing. This way we wouldn’t need to highlight the message in a particular fashion.

What do you think?

markusjura commented:
Unit tests are failing due to not updated tests.

huntc commented:
I had problems running all tests due to the docker low on mem error throwing things out. I was being lazy. Apologies.

markusjura committed:
Fix host unit tests

Fix the unit tests of the `host.py` according to the changes of the previous commit. Also does mark the message `Whoops. Network address aliases are required so that the sandbox can operate as a cluster of machines.` as a headline.

markusjura commented:
Fixed the unit tests. Also, the message `Whoops. Network address aliases are required so that the sandbox can operate as a cluster of machines.` is not marked as a headline anymore.

**New output**

```
|------------------------------------------------|
| Starting ConductR                              |
|------------------------------------------------|
Whoops. Network address aliases are required so that the sandbox can operate as a cluster of machines.

Please run the following and then try your command again:

sudo ifconfig lo0 alias 192.168.10.1 255.255.255.0
```
